projectiles/NTacticalMissile1_HighArc/NTacticalMissile1_HighArc_proj.bp

```diff
@@ -59,15 +59,15 @@ ProjectileBlueprint {
         HelpText = 0,
     },
     Physics = {
-        Acceleration = 0,
+	Acceleration = 3,
```

Comment:
you changed the style from spaces to tabs here. Please keep it consistent with the rest of the code. (tabs -> spaces)